File: WEEK 1/C++/Calculate Area of a Circle.cpp
#include <iostream>
using namespace std;

int main() {
    const double Pi = 3.141593;
    double radius;

    cout << "Enter the radius of the circle: ";
    cin >> radius;

    double area = Pi * pow(radius, 2);
    printf("Area of the circle: %.2f\n", area);

    return 0;
}

File: WEEK 1/C#/Calculate Area of a Circle.cs
using System;

class Program {
    static void Main() {
        const double Pi = 3.141593;

        Console.Write("Enter the radius of the circle: ");
        double radius = double.Parse(Console.ReadLine());

        double area = Pi * Math.Pow(radius, 2);
        Console.WriteLine($"Area of the circle: {area:F2}");
    }
}